app/Nova/Resources/Resource.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Nova\Resources;
+
+use Illuminate\Database\Eloquent\Builder;
+use Laravel\Nova\Http\Requests\NovaRequest;
+use Laravel\Nova\Resource as NovaResource;
+
+abstract class Resource extends NovaResource
```

Comment:
What added value does this abstract class have exactly? It seems like it overrides some methods but either implements them exactly the same way (`indexQuery`) or just calls the parent method (`detailQuery`, `relatableQuery`)

Reply:
Indeed, that can be removed and added again if needed. It's part of the default scaffolding when running `nova::install`

So I removed that class for now.